stories/accordion/index.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withDocs } from 'storybook-readme';
+import { withKnobs, text, boolean } from '@storybook/addon-knobs';
+
+import AccordionReadme from 'docs/accordion.md';
+import { Accordion, Typography } from 'src/shared-components';
+import { SPACING } from 'src/constants';
+
+import DefaultAccordion from './defaultAccordion';
+
+const stories = storiesOf('Accordion', module);
+
+stories.addDecorator(withKnobs);
+
+stories.add(
+  'Usage',
+  withDocs(AccordionReadme, () => (
+    <React.Fragment>
+      <div css="text-align: left;" >
+        <DefaultAccordion />
+        <Typography.Heading css={`padding: ${SPACING.base} 0 ${SPACING.small};`}>
+          With Knobs
+        </Typography.Heading>
+        <div css="width: 400px;">
+          <Accordion.Container>
+            <Accordion
+              title={
+                <Accordion.Content>
+                  <Accordion.Truncate>
+                    {text('Title text', 'Title')}
+                  </Accordion.Truncate>
+                </Accordion.Content>
+              }
+              onClick={() => {}}
```

Comment:
we can use an "action" to log the click similar to this: https://github.com/PocketDerm/radiance-ui/blob/644500fc6ec02eb3d41af1168a4bafcd1185096f/stories/radioButton/index.js#L5